internal/monitors/collectd/kafka/kafka.go

```diff
@@ -39,7 +39,7 @@ func init() {
 	}
 	defaultMBeans = defaultMBeans.MergeWith(genericjmx.DefaultMBeans)
 
-	monitors.Register(monitorType, func() interface{} {
```

Comment:
Make sure the `monitorType` const is consistently deleted so it isn't duplicated with the value in `monitorMetadata`.